receive.c

```diff
@@ -34,6 +34,8 @@
 static int main_loop = 1;
 static int sockfd = 0;
 static char *recvBuff = NULL;
+char **filters = NULL;
```

Comment:
Dunno why GitHub says that this diff isn't outdated, removed the second definition.